scripts/zkai_native_seq32_attention_mlp_adapter_variant_selector_gate.py

```
@@ -0,0 +1,763 @@
+
+def validate_payload(payload: dict[str, Any]) -> None:
+    expected_top = {
+        "schema": SCHEMA,
+        "decision": DECISION,
+        "result": RESULT,
+        "issue_hint": ISSUE_HINT,
+        "claim_boundary": CLAIM_BOUNDARY,
+    }
+    for key, expected in expected_top.items():
+        if payload.get(key) != expected:
+            raise NativeSeq32AdapterVariantSelectorGateError(f"{key} drift")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)